bundles/org.openhab.binding.knx/doc/dpt.txt

```diff
@@ -46,8 +46,8 @@ MainType: 3
 3.008: DPT_Control_Blinds values: 0 = up 1 = down
 
 MainType: 4
-4.001: DPT_Char_ASCII
-4.002: DPT_Char_8859_1
+unsupported 4.001: DPT_Char_ASCII
+unsupported 4.002: DPT_Char_8859_1
```

Comment:
Just a side-note, not related and out of scope: why don't we support that?

Reply:
Calimero does not support it...